helmfile.d/snippets/defaults.yaml

```diff
@@ -1266,21 +1266,21 @@ environments:
             resources:
               limits:
                 cpu: "1"
-                memory: 1Gi
+                memory: 192Mi
               requests:
                 cpu: 50m
-                memory: 512Mi
+                memory: 64Mi
```

Comment:
Would it not be better to set the requests equal with the limit, for increasing the QoS class (https://cloudnative-pg.io/documentation/1.24/resource_management/)?